x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { useSelector } from 'react-redux';
+import { applyMatrix3 } from '../lib/vector2';
+import { Vector2 } from '../types';
+import * as selectors from '../store/selectors';
+
+export const ProcessEventDot = styled(
+  React.memo(
+    ({
+      className,
+      worldPosition,
+      processEvent,
+    }: {
+      className?: string;
+      worldPosition: Vector2;
+      processEvent: any;
```

Comment:
can you change this to `unknown` or some more specific type?

Reply:
Changed it to `ProcessEvent` type.